CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+## 1.20.0-rc1 (September 19, 2024)
+
+SECURITY:
```

Comment:
I think we are missing the changelog for this PR: https://github.com/hashicorp/consul/pull/21729/files
But possibly this was after code freeze?

Reply:
Doesn't look like it was backported into 1.20 so it didn't get included in the branch. @zalimeni should it have been?

Reply:
I'm going to go forward with the merge, but if we need to backport it into the branch let me know